CONTRIBUTING.md

```diff
@@ -28,3 +28,51 @@ Members of the Contributors team are encouraged to review pull requests that
 have already been reviewed, and pull request contributors are encouraged to seek
 multiple reviews.  Reviews from anyone not on the Contributors team are always
 appreciated and encouraged!
+
+## Reviewing Pull Requests
+
+All mlpack contributors who choose to review and provide feedback on Pull
+Requests have a responsibility to both the project and the individual making
+the contribution. 
+
+Reviews and feedback must be
+[helpful, insightful, and geared towards improving the contribution](
+  https://www.youtube.com/watch?v=NNXk_WJzyMI).
+If there are reasons why you feel the PR should not be merged, explain
+what those are. Do not expect to be able to block a Pull Request from advancing
+simply because you say "No" without giving an explanation. Be open to having
```

Comment:
```
Do not expect to be able to block a Pull Request from advancing
simply because you say "No" without giving an explanation.
```

This is a little bit redundant, we could probably keep it a bit shorter by dropping this sentence if you like.  I think the rest of the document does make it pretty clear that explanations and help are the way to go, not blocking PRs without saying more. :)

Reply:
Removed